pkg/replication/replication.go

```diff
@@ -235,7 +241,7 @@ func (r replication) Enact() {
 		select {
 		case <-done:
 			doneIndex++
-			r.logger.Infof("Nodes done (or timed out): %v", doneIndex)
+			r.logger.Infof("Nodes done, timed out, or cancelled: %v", doneIndex)
```

Comment:
oh, this explains why you did the `defer` change below, although I'm not sure why timed out and canceled nodes would be lumped together